ouroboros-consensus/src/unstable-consensus-testlib/Test/QuickCheck/Extras.hs

```diff
@@ -0,0 +1,16 @@
+module Test.QuickCheck.Extras (
```

Comment:
Perhaps these functions could be upstreamed to QC eventually ...

Reply:
Indeed.